docs/manual.rst

```diff
@@ -2301,7 +2301,31 @@ filters.typogrify_sans_widont
    Same as typogrify without the widont filter
 
 filters.typogrify_custom
-    Run typogrify with a custom set or filters. Takes ``typogrify_filters`` (a list of callables) and ``ignore_tags`` (defaults to None).
+    Run typogrify with a custom set of filters or ignored tags. Takes ``typogrify_filters``
+    (a list of callables) or ``ignore_tags`` (a list of HTML tags to ignore). These must be
```

Comment:
> (a list of HTML tags to ignore)

Not quite, typogrify takes elements, classes (with .), IDs (with #), and constructions like element#id. This should be described more precisely.